src/components/UniversalTable/UniversalTable.tsx

```diff
@@ -0,0 +1,30 @@
+import {  ReactNode } from 'react';
+import classes from './UniversalTable.module.scss';
+
+type UniversalTableProps<D> = {
+  data: D[]
+
+  headers: {
+    title: string;
+    property: keyof D;
+  }[]
+}
+
+const UniversalTable = <T,>({data, headers}:UniversalTableProps<T>):ReactNode => {
+  return (
+    <>
```

Comment:
You don't need a fragment here as long as you return single div with children

Reply:
fragment has been removed